### PR TITLE
Enable Travis

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

**Comment by eile** [comment]
ping

**Comment by tribal-tec** [comment]
No install of Qt & LibJPEGTurbo to cover more code to build?

**Comment by eile** [comment]
are not mentioned in PORT_DEPENDS... adding

File: .travis.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ env:
 before_install:
   - brew update
   - brew outdated cmake || brew upgrade cmake
-  - brew install cppcheck doxygen ninja
```
Comment (tribal-tec) [inline .travis.yml]: qt5 won't work that, it needs an export on CMAKE_PREFIX_PATH. Check according projects :)

**Comment by tribal-tec** [comment]
+1

**Pull request merged**